### PR TITLE
[Merged by Bors] - feat: port RingTheory.WittVector.Frobenius

### PR DESCRIPTION
**Pull request by xroblot**
---

- [x] depends on: #5284 

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by semorrison** [comment]
I'm adding `help-wanted` here.

**Comment by semorrison** [comment]
This PR/issue depends on:
* leanprover-community/mathlib4#5284
By **[Dependent Issues](https://github.com/z0al/dependent-issues)** (🤖). Happy coding!

**Comment by semorrison** [comment]
This PR/issue depends on:
* ~~leanprover-community/mathlib4#5284~~
By **[Dependent Issues](https://github.com/z0al/dependent-issues)** (🤖). Happy coding!

**Comment by jcommelin** [comment]
I merged master. `ghost_calc` is now available.

File: Mathlib/RingTheory/WittVector/Frobenius.lean [hunk 1]
```diff
@@ -67,213 +65,210 @@ open scoped BigOperators
...
 
-/-- `frobenius_fun` is tautologically a polynomial function.
+/-- `frobeniusFun` is tautologically a polynomial function.
 
-See also `frobenius_is_poly`. -/
-@[is_poly]
-theorem frobeniusFun_isPoly : IsPoly p fun R _Rcr => @frobeniusFun p R _ _Rcr :=
-  ⟨⟨frobeniusPoly p, by intros; funext n; apply coeff_frobenius_fun⟩⟩
+See also `frobenius_isPoly`. -/
+instance frobeniusFun_isPoly : IsPoly p fun R _Rcr => @frobeniusFun p R _ _Rcr :=
```
Comment (jcommelin) [inline Mathlib/RingTheory/WittVector/Frobenius.lean:234]:
```suggestion
-- Porting note: replaced `@[is_poly]` with `instance`.
instance frobeniusFun_isPoly : IsPoly p fun R _Rcr => @frobeniusFun p R _ _Rcr :=
```

File: Mathlib/RingTheory/WittVector/Frobenius.lean [hunk 2]
```diff
@@ -290,8 +285,7 @@ theorem ghostComponent_frobenius (n : ℕ) (x : 𝕎 R) :
 variable (p)
 
 /-- `frobenius` is tautologically a polynomial function. -/
-@[is_poly]
-theorem frobenius_isPoly : IsPoly p fun R _Rcr => @frobenius p R _ _Rcr :=
+instance frobenius_isPoly : IsPoly p fun R _Rcr => @frobenius p R _ _Rcr :=
```
Comment (jcommelin) [inline Mathlib/RingTheory/WittVector/Frobenius.lean:288]:
```suggestion
-- Porting note: replaced `@[is_poly]` with `instance`.
instance frobenius_isPoly : IsPoly p fun R _Rcr => @frobenius p R _ _Rcr :=
```

**Comment by jcommelin** [comment]
bors d+

**Pull request closed without merging**